README.md

```diff
@@ -83,9 +83,9 @@ contract MyContract {
 - `getSource({ network })` Return `abi` and `bytecode` for a contract `source`.
 - `getSuspensionReasons({ code })` Return mapping of `SystemStatus` suspension codes to string reasons.
 - `getStakingRewards({ network })` Return the list of staking reward contracts available.
-- `getHassets({ network })` Return the list of hassets for a network.
+- `getSynths({ network })` Return the list of zassets for a network.
```

Comment:
Why "getSyncths" ? Shouldn't be "getZassets"?

Reply:
Yeah, I'll change it.